articles/references/domain-designer/associations/associations.md

```diff
@@ -42,100 +40,100 @@ This can be achieved by setting the association like this:
 
 ### Aggregation Association
 
->Aggregation relationship
+> Aggregation relationship
 >
->1. When representing a software or database relationship, e.g. car model engine ENG01 is part of a car model CM01, as the engine, ENG01, may be also part of a different car model.[8]
->2. When the container is destroyed, the contents are usually not destroyed, e.g. a professor has students; when the professor dies the students do not die along with them.
-
-*[Wikipedia - Composition](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
+> 1. When representing a software or database relationship, e.g. car model engine ENG01 is part of a car model CM01, as the engine, ENG01, may be also part of a different car model.[8]
+> 2. When the container is destroyed, the contents are usually not destroyed, e.g. a professor has students; when the professor dies the students do not die along with them.
+>
+> *- [Wikipedia](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
 
-![Aggregation Association](images/association-aggregate-professor.png)
+![How an Aggregation Association appears in UML and Intent Architect's Domain Designer](images/association-aggregate-professor.png)
 
-This can be achieved by setting the association like this:
+An Aggregation Association be achieved by setting the properties of the Association as follows:
 
 |Source End|Target End|
 |-|-|
 |⬜ Navigable|✅ Navigable|
 |✅ Is Nullable|⬜ Is Nullable|
 |⬜ Is Collection|✅ Is Collection|
 
-## Multiplicity Types
-
->Multiplicity is a definition of cardinality - i.e. number of elements - of some collection of elements by providing an inclusive interval of non-negative integers to specify the allowable number of instances of described element.
+## Multiplicity types
 
-*[UML Diagrams](https://www.uml-diagrams.org/multiplicity.html)*
+> Multiplicity is a definition of cardinality - i.e. number of elements - of some collection of elements by providing an inclusive interval of non-negative integers to specify the allowable number of instances of described element.
+>
+> *- [uml-diagrams.org](https://www.uml-diagrams.org/multiplicity.html)*
```

Comment:
I realize this is inconsistent with above where we say `Wikipedia` while here we use a domain name. I felt that Wikipedia is well known enough that everyone knows it's a website so it can just be referred to by name, while `uml-diagrams.org` is not generally well known and hence it makes sense to refer to it by its domain name.

Happy to have other opinions though.

Reply:
That's fine. I don't think any of us have every heard of the site either so adding the domain name should be fine.